modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc

```diff
@@ -0,0 +1,201 @@
+// tag::description[]
```

Comment:
@kmendiratta-crm  this is the reusable page. Which sections aren't applicable to BYOK?